appmap/django.py

```diff
@@ -133,25 +133,32 @@ def request_params(request):
     return values_dict(params.lists())
 
 def normalize_path_info(path_info, resolved):
-    # Prior to django 3.2, there doesn't appear to be a good way to
-    # determine whether the pattern that matched a request was a path
-    # or a re_path. To figure it out, just try to use the route as a
-    # regex. If that fails, assume it's a path, and use django's
-    # internal parser to turn it into a regex.
-    regex = resolved.route
-    match = re.match(regex, path_info[1:])
-    if not match:
-        regex = _route_to_regex(resolved.route)[0]
+    if not resolved.kwargs:
+        # No kwargs mean no parameters to normalize
+        return path_info
+
+    # Get the current resolver. Look through its cache of reverse
+    # route mappings and find the regex that matches the current
+    # request. The route mappings are tuples of the form:
+    #   (format string, regex, args, kwargs)
+    resolver = get_resolver()
+    regex = None
+    for _,regex,_,_ in resolver.reverse_dict.values():
```

Comment:
`resolver.reverse_dict` is a multi valued dict, so `values()` will skip some items if there are views which match more than one pattern. If you use `user_view` instead of a separate `included_user_view` in the test below you can reproduce this issue.

Reply:
Ok, thanks, I'll investigate.